lib/publiccloud/instance.pm

```diff
@@ -476,7 +481,9 @@ sub wait_for_ssh {
 
         # Finally make sure that SSH works
         while (($duration = time() - $start_time) < $args{timeout}) {
-            $exit_code = $self->ssh_script_run(cmd => "true", username => $args{username}, timeout => $args{timeout} - $duration);
+            # After the instance is resumed from hibernation the SSH can freeze
+            my $ssh_opts = $self->ssh_opts() . ' -o ControlPath=none -o ConnectTimeout=10';
+            $exit_code = script_run("ssh -v $ssh_opts -l $args{username} $self->{public_ip} true", timeout => $args{timeout} - $duration);
```

Comment:
```suggestion
            $exit_code = script_run("ssh -v $ssh_opts -l $args{username}@$self->{public_ip} true", timeout => $args{timeout} - $duration);
```

hm usually I call ssh command like this , is notation w/o `@` is supported or it was just typo ?